yarn.lock

```diff
@@ -210,6 +210,29 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@babel/core@npm:^7.17.10":
```

Comment:
Where does all the new dependencies come from?

Reply:
It uses `babel-loader` to process TS files